jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudformation/json/JsonFormatViolationInspectionTest.kt

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudformation.json
+
+import com.intellij.testFramework.runInEdtAndWait
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Rule
+import org.junit.Test
+import software.aws.toolkits.jetbrains.utils.rules.JavaCodeInsightTestFixtureRule
+
+class JsonFormatViolationInspectionTest {
```

Comment:
Should add a test for no problems = no highlighting

Reply:
Easy to add, sure.

Reply:
Added